### ISSUE TITLE
6.9 Medic Feedback

### ISSUE THREAD

**Issue by RhynxSyn** [Reagy/TF2Classic-KO-Custom-Weapons#332]
**TL;DR**
Most of the hitscan weapons are decent, more support weapons (specifically primary) would be good.
Spritzenwerfer too spammy, Guardian Angel's shield should be changed to burst heal, Hydropump too situational, Kralle's stats don't mix together and just kinda suck and Blitz-Storm looks fucking EPIC.
Change SAW's crits on kill to haste on kill, revert the Syrup's boost type and please do something with the Festive Twig :(.
(The rest of this post is basically just extra feedback along with providing reasoning for the stuff above)

**Medical Malpractice + Vogeljäger**

THESE WEAPONS....... They're actually kinda good. Medical Malpractice deals around 50-60 while Vogel is 60-70 which is alright for a Medic shotgun. The inaccuracy and high pellet count makes it so you have to get closer, making classes like Soldiers and Pyros have an easier time killing you but it is what it is.

**Ol' Thompson**

This weapon is kinda alright, but I don't really think it's that good. The close range damage doesn't actually really do much, as the other normal SMGs like death dealer and Heerman's also do around the same damage up close. Along with this, making Medic slow is just a bad idea in general. I think it'd be better if either the slow when active was removed or the close range damage was buffed.

**Surgeon's Sterling** 

This SMG is the worst one. It's not good at all... It does like +2 more damage than the death dealer for being more slower and less accurate. The increased self-healing feels minimal aswell, just not a good weapon overall. Either this could just have it's stats buffed (mainly the damage, maybe even making it 50% so it deals 15 damage per hit?) or maybe having some sort of gimmick stat like faster firing speed on kill to make it a mini snowball weapon.

**Heerman's + Death Dealer**

These are SUPER good! The only main issue is that the Heerman's clip size nerf isn't enough and overshadows the Death Dealer most of the time. Maybe if the Dealer had a faster firing speed it'd be more up to par compared to it, but right now the Heerman's seems like the best SMG to use.

**Practitioner's Pride**

This thing fell off so hard, which makes it even worse since the Super Syringe Gun exists. Why take out a light class in 4 direct bursts with the Practitioner's when you can take them out faster with 2 bursts with the SSG that don't have to hit all syringes in the burst?

Since it used to be a support syringe gun and I saw a biohazard symbol on it it gave me an idea of a rework. Maybe it could be like a Medic version of Spy's Saboteur, being a toxin dart gun that can self heal overtime. It'd also give Medic more support to his primaries (which I'd prefer if he had more of) and could also add counterplay to pocketing.

**Degrader**

Degrader is definitely an improvement, but compared to other utility primaries it's a bit lackluster, especially due to the MFD time. I think it'd be better if it was buffed to 5 seconds, as compared to the old practitioner's it used to fire faster and lasted longer, so it's definitely weaker in the MFD part.

**Lightning-Storm**

I fucking LOVE this! This weapons so fucking AWESOME! Uber is like REALLY important and all so having this work as a utility and a decent weapon that hits easily is REALLY good for uber gaining.

**Guardian Angel**

This medigun is alright but my one worry is the Ubercharge. I being a 80 health shield is alright, but the main problem is that it also bypasses any damage aslong as it happens in one go. The shield effectively has infinite HP as long as it's in one shot, so stuff like backstabs can be prevented.

Having it so the Guardian Angel gives a burst healing of 80-100 (or maybe just giving 50% of a patients max health) would make it so that you can give patients a burst healing to prevent them from dying, but not from tanking a crit rocket when they should've died either way.

**Kralle**

I don't really like the Kralle mostly because:

1. The slow movement and switch speed makes Medic easier to be dived on. The slow movement speed especially effects who you can uber and when.
2. The less uber on non-overhealed players and more on overhealed players encourages either Medic to play lazily in the backlines or to just pocket someone to get that overheal bonus.
3. It SEEMS like it's supposed to have a tankier/defensive kind of playstyle with the longer uber and health regen, but both stats are pretty minimal and don't add much to the weapon overall.

**Hydropump**

On paper this weapon is fine, but in reality you aren't going to heal multiple people in a wide range unless in a very specific scenario, which is usually when you're in a group in a choke in which a stock uber or kritz would be better in that situation for pushing. I think it'd be better if the healing was more narrow but healed more, or the spray just had a more wider range overall.

The uber isn't really that effective, as while it is cool to just burst heal a buncha guys and push in, it takes too long and isn't really worth it in most instances, let alone actually getting more than 2 people healed.

**Spritzenwerfer**

This gun's actually pretty good now, though I have a few small issues with it.

1. The syringes don't have a drop-off, so they have infinte range. This leads to easy uber building and making it easy to spam from long ranges.
2. Slow on Medic is REALLY BAD!!! Even if it is Battle Medic, I think it'd be better if the less speed on wearer was shifted and added on to the "while active" nerf already.
3. Similarly to above, the way the Sprtizenwerfer works, mainly the slower move/switch speed and lack of self-survivability makes it so that the Sprtizenwerfer doesn't really have much wiggle room for loadout choice, making it so you'll most likely pick the Self-Care or Reconstruction Syrup

**Blitz-Storm**

Not an official weapon yet but I wanted to say this weapon seems REALLY cool and I'd love to test it out. It gives me the same vibe of the old toxic waste in which it was a medigun that healed and dealt damage in a "self-defense scenario". It seems like it's not going to be a more "battle medic" based weapon, but giving it some sort of buff on enemy hit as a deterrance or even giving the medic uber on hit so that he won't waste time damaging someone would be great. 

I had suggested to Noclue a few days ago when making this post that the toxic waste would do mark for death for one second, making it so keeping your beam on an enemy would give more effective value to allies, maybe that would work well here too? I could see this medigun working really well with more utility based weapons, both melee and primary.

**SAW**

I don't really like this weapon at all. It puts the Medic at high risk with a slow melee that encourages you to charge INTO the enemy for crits that are situational and might not even be used before you die.

Personally I think it'd be better if the SAW got haste on kill instead, as it'd be better for these reasons:

1. It gives the Medic a utility that allows him to use the SAW to escape efficiently after a kill, similarly to the Blood Donor.
2. It allows the SAW to be able to be used effectively after a kill, having a similar "cut down each enemy after a kill" kind of feel similar to the Cut Above.
3. It would still increase the amount of damage output you can get, and would even improve it in combonation with other weapons like the Doctor's Self-Care. It'd also be able to be used more when switching weapons, as the haste will help put the SAW away faster.

Also make it bypass resistances cuz crit resistance exists lmao...

**Reconstruction Syrup**

I'm not a fan of this weapon, but I can see how healing out-of-combat can help you get back to the frontlines quicker, along with it being really good with the surgical precision and spritzenwerfer.

I don't really like the boost that much, as the medkit is just better all round for healing as it provides a burst heal that is more likely to save people from dying compared to the syrups "death prevention" boost. I think people would prefer to use it more if it had the old uber boost back, as I found it way more useful compared to the current boost.

**Festive Twig**

PLEASE DO SOMETHING WITH THIS!!! THERE'S NO POINT TO THIS!!! This applies especially to Medic as the Blood Donor exists. Make it do something please I beg. One main idea I thought of is that you receive more healing, but don't have any regen. Because of it dropping health kits on kill I could see this being reworked into a good support weapon, maybe even making the medkit on kill stat global so Battle Medic can have an alternative to the other survivability melees.
